.circleci/config.yml

```diff
@@ -51,7 +51,18 @@ jobs:
           key: v1-deps-server-cache-{{ checksum "./server/yarn.lock" }}
       - run:
           name: Test server
-          command: cd server && yarn && yarn test
+          command: cd server && yarn && yarn
```

Comment:
I wonder if there is a way we can avoid using yarn or npm?
Nothing to change, but let's explore this on a later date.